test/tests/selftest/pbs_microsecondlogging.py

```diff
@@ -41,32 +41,33 @@
 
 class TestMicrosecondLogging(TestSelf):
     """
-    Test pbs microsec logging
+    Test log_match() works when start and endtime
+    values are in microseconds
     """
 
-    def setup_microsecondlogging(self, hostname=None, highrestimestamp=1):
+    def switch_microsecondlogging(self, hostname=None, highrestimestamp=1):
         """
         Set micro second logging in pbs.conf
         """
-        du = DshUtils()
+        self.du = DshUtils()
```

Comment:
You shouldn't have to initialize 'du', PTL already provides you with one, just try using 'self.du' without initializing it.